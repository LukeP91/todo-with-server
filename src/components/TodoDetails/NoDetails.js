import { withStyles } from '@material-ui/core/styles'
import Paper from '@material-ui/core/Paper'
import PropTypes from 'prop-types'
import React from 'react'
import Typography from '@material-ui/core/Typography'

const styles = theme => ({
  paper: {
    padding: theme.spacing.unit * 2,
    textAlign: 'center',
    color: theme.palette.text.secondary,
  },
})

const NoDetails = ({ classes }) => (
  <Paper className={classes.paper}>
    <Typography align="left" gutterBottom variant="body1">
      Select todo from the list on the right to see details.
    </Typography>
  </Paper>
)

NoDetails.propTypes = {
  classes: PropTypes.object.isRequired,
}

export default withStyles(styles)(NoDetails)
