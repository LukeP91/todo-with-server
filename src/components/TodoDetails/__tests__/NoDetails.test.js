import React from 'react'
import renderer from 'react-test-renderer'

import NoDetails from '../NoDetails'

describe('NoDetails', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoDetails />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
