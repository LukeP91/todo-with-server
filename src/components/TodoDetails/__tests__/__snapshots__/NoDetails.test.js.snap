// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoDetails renders correctly 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation2-6 MuiPaper-rounded-3 NoDetails-paper-1"
>
  <p
    className="MuiTypography-root-29 MuiTypography-body1-38 MuiTypography-gutterBottom-46 MuiTypography-alignLeft-41"
  >
    Select todo from the list on the right to see details.
  </p>
</div>
`;
